workflows_api/runtime/src/auth.py

```diff
@@ -24,7 +19,13 @@ def get_settings() -> config.Settings:
 
 
 def get_jwks_url(settings: config.Settings = Depends(get_settings)) -> str:
-    return settings.jwks_url
+    import boto3
+    import json
+    client = boto3.client("secretsmanager")
+    response = client.get_secret_value(SecretId=settings.workflows_client_secret_id)
+    secrets = json.loads(response["SecretString"])
+
+    return f"https://cognito-idp.{secrets['aws_region']}.amazonaws.com/{secrets['userpool_id']}/.well-known/jwks.json"
```

Comment:
instead of asking for `jwks_url` env var, derive from secrets

Reply:
auth secrets doesn't return `SecretString` anymore but is a dict. Will need:
```
aws_region = response['aws_region']
userpool_id = response['userpool_id']

return f"https://cognito-idp.{aws_region}.amazonaws.com/{userpool_id}/.well-known/jwks.json"
```